### PR TITLE
fixes#2394-developmental terms

### LINKED ISSUE #2394: NTR: developmental terms
**Comment:** The following terms are needed to indicate the origin of the tissues used establish cell lines annotated in Cellosaurus. Many thanks

**Preferred term label:** aorta gonad mesonephros region
**Synonyms**
aorta-gonad-mesonephros
embryonic aorta gonad mesonephros
**Definition** The aorta-gonad-mesonephros region is an area derived from splanchnopleura mesoderm identified in embryonic humans, mice, and non-mammalian vertebrates such as birds and zebrafish. It contains the dorsal aorta, genital ridges and mesonephros and lies between the notochord and the somatic mesoderm, extending from the umbilicus to the anterior limb bud of the embryo. This region plays an important role in embryonic development, being the first autonomous intra-embryonic site for definitive haematopoiesis. [https://en.wikipedia.org/wiki/Aorta-gonad-mesonephros]
**Parent term** mesoderm UBERON:0000926
**NOTE** derived from splanchnopleure UBERON:0004873
**Your nano-attribution (ORCID)** 0000-0002-0819-0473

**Preferred term label:** ventral presumptive midbrain
**Synonyms** ventral mesencephalon
**Parent term** presumptive midbrain UBERON:0009616
**NOTE** This term will be use to indicate the origin of the cell line: NGC-407 (CVCL_DF59) established from "ventral mesencephalon of a 7-week-old human embryo" PubMed=17555745
**Your nano-attribution (ORCID)** 0000-0002-0819-0473

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2394 -developmental terms

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224854,6 +224854,23 @@ relationship: composed_primarily_of CL:0000057 ! fibroblast
...
+id: UBERON:8480035
+name: aorta-gonad-mesonephros region
+def: "Embryonic region that contains the dorsal aorta, genital ridges and mesonephros. It lies between the notochord and the somatic mesoderm, extending from the umbilicus to the anterior limb bud. This region is the first autonomous embryonic site for definitive haematopoiesis." [https://en.wikipedia.org/wiki/Aorta-gonad-mesonephros]
+synonym: "AGM region" EXACT [PMID:16480451]
+is_a: UBERON:0005291 ! embryonic tissue
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-10-10T14:07:07Z xsd:dateTime
+
+[Term]
+id: UBERON:8480036
```
Comment (shawntanzk) [inline src/ontology/uberon-edit.obo:224867]: needs a definition

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224854,6 +224854,23 @@ relationship: composed_primarily_of CL:0000057 ! fibroblast
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-03T14:38:17Z xsd:dateTime
 
+[Term]
+id: UBERON:8480035
+name: aorta-gonad-mesonephros region
+def: "Embryonic region that contains the dorsal aorta, genital ridges and mesonephros. It lies between the notochord and the somatic mesoderm, extending from the umbilicus to the anterior limb bud. This region is the first autonomous embryonic site for definitive haematopoiesis." [https://en.wikipedia.org/wiki/Aorta-gonad-mesonephros]
```
Comment (shawntanzk) [inline src/ontology/uberon-edit.obo:224860]: I don't know if im comfortable wit the wording "contains the dorsal aorta, genital ridges and mesonephros" as I assume it contains structures that become those regions? correct me if I'm wrong - in which case the structure does not technically contain it

Comment (shawntanzk) [reply src/ontology/uberon-edit.obo:224860]: "This region is the first autonomous embryonic site for definitive hematopoiesis" 
can this be interpreted as it having the axiom "'site_of' some hemopoiesis"? 
I ask as having that axiom will classify this as a hematopoietic tissue which I assume it is

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224854,6 +224854,23 @@ relationship: composed_primarily_of CL:0000057 ! fibroblast
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-03T14:38:17Z xsd:dateTime
 
+[Term]
+id: UBERON:8480035
```
Comment (shawntanzk) [inline src/ontology/uberon-edit.obo:224858]: I think we need to also take a look at [UBERON:0005805](http://purl.obolibrary.org/obo/UBERON_0005805) (dorsal aorta), [UBERON:0005294](http://purl.obolibrary.org/obo/UBERON_0005294) (gonadal ridge), and [UBERON:0000080](http://purl.obolibrary.org/obo/UBERON_0000080) (mesonephros) - all of which already have develops from axioms. These should all be coherent with this, and develops_from axioms need to be added where appropriate to link this term up correctly

**Review by shawntanzk — CHANGES_REQUESTED** [review]
I've left some comments in line, but development stuff is really difficult and I would be more comfortable if you had someone with more expertise review this. Hope that helps :)

File: src/ontology/uberon-edit.obo (hunk 2 above)
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224860]: @shawntanzk I think that the region contains these embryonic structures: embryonic aorta, genital ridges and mesonephros (all embryonic structures)
@shawntanzk OK to add: axiom "'site_of' some hemopoiesis"

**Comment by shawntanzk** [comment]
@cerivs sorry to bother, but I do not have enough knowledge to review this and the requester pointed to you as someone who might be able to help. Would you mind helping us take a look please? thanks in advance :)

**Comment by paulacalipho** [comment]
Hi @shawntanzk I made some modifications to this pull request and ask for another reviewer also

**Pull request closed without merging**